src/code-editor/styles.scss

```diff
@@ -177,6 +175,30 @@
   &--errors {
     /* used in test-utils */
   }
+
+  /* stylelint-disable plugin/no-unsupported-browser-features */
```

Comment:
Please check if this is still needed after  https://github.com/cloudscape-design/components/pull/2340

Reply:
Confirmed, this is still required.

Reply:
But this could be a smaller scope exception, because `@supports` does not need this anymore

Reply:
Oh, I see. Sure we can do that.